migrations/erc2362.addresses.json

```diff
@@ -7,7 +7,6 @@
             "OmgBtcPriceFeed": "0xffffffffffffffffffffffffffffffffffffffff",
             "OmgEthPriceFeed": "0xffffffffffffffffffffffffffffffffffffffff",
             "OmgUsdtPriceFeed": "0xffffffffffffffffffffffffffffffffffffffff",
```

Comment:
```suggestion
            "OmgUsdtPriceFeed": "0xffffffffffffffffffffffffffffffffffffffff"
```

Reply:
Actually I can add this change when deploying the pricefeeds and adding the addresses

Reply:
Thanks!